src/wallet/rpc/spend.cpp

```diff
@@ -1048,6 +1055,15 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
             coin_control.m_signal_bip125_rbf = options["replaceable"].get_bool();
         }
         SetFeeEstimateMode(*pwallet, coin_control, conf_target, options["estimate_mode"], options["fee_rate"], /*override_min_fee=*/false);
+
+        // Prepare new outputs by creating a temporary tx and calling AddOutputs().
+        if (!options["outputs"].isNull()) {
```

Comment:
b2997036b84ed84404601cc4f5e088a9c7a87abd: sorry for not testing this, but if I use an empty array for outputs, is it going to send everything to the change address? What if there was no change address? Should we require at least 1 output if this field is present?

Reply:
I did a quick modification of the existing test to test this, and it looks like if someone were to use an empty array for outputs, then the resulting fee-bumped transaction would have the same outputs as the transaction it is replacing, which is what I would expect to happen from looking at the code. I think it would be useful to at least mention somewhere that if no outputs are provided but the field is still present, then the original outputs are used.

Reply:
This was mentioned in the RPC doc above.

However I think it is a bit confusing to have an empty array of outputs be synonymous with not providing the option at all. I think it would be better to just throw an error in that case. Checking for an empty outputs array could be done in `AddOutputs` as well so that check will also be done for `createpsbt` and `createrawtransaction` as it doesn't make sense to make a transaction with no outputs.

Reply:
I still think this needs to be addressed, either implementing the suggestion or a comment defending this decision to allow an empty outputs array.

Reply:
Throwing an error when an empty array is used with this argument make sense. Error message could be something like: `"Invalid parameter, output argument must not be an empty array"`